src/Symfony/Component/DomCrawler/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.4
+---
+
+ * Added `Crawler::innerText` method.
```

Comment:
```suggestion
 * Add `Crawler::innerText` method.
```

Reply:
Past tense is consistent with every other entry in this file. Why should this be changed now?

Reply:
We changed it some time ago.

For reference:
https://symfony.com/doc/current/contributing/code/conventions.html#writing-a-changelog-entry